calcom/cal.com
# Address feedback of feedback feature

## What does this PR do?



When submitting feedback the menu item now closes on a single click of the submit button.

https://www.loom.com/share/1405d559abe7436aa32ddb6e000a54df

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Submit feedback
	- It should only take one click and the feedback box should close

## Checklist



- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
